gui/src/renderer/components/NotificationBanner.tsx

```diff
@@ -213,6 +213,7 @@ export class NotificationBanner extends Component<
     if (prevProps.visible !== this.props.visible) {
       // enable drawer-like animation when changing banner's visibility
       this.setState({ contentPinnedToBottom: true }, () => {
+        // eslint-disable-next-line @typescript-eslint/no-floating-promises
         this.animateHeightChanges();
```

Comment:
You `await` in here but don't `await` below. 

Maybe just wrap this call into something like:

```
animateHeightChangesHelper() {
   this.animateHeightChanges().then(() => {})
      .catch((_error) => {})
}
```

Reply:
There's also an eslint rule for not having empty functions which would complain about this.